src/azure-cli/azure/cli/command_modules/eventhubs/custom.py

```diff
@@ -621,7 +621,8 @@ def cli_remove_appgroup_policy(client, resource_group_name, namespace_name, appl
     from azure.cli.core import CLIError
 
     appGroup = client.get(resource_group_name, namespace_name, application_group_name)
-
+    logger.warning(
+        'the policy remove policy cmdlets will do removals based on policy name')
```

Comment:
The sentence sounds awkward to me...
```suggestion
        'This operation will do removals based on policy name')
```